.gemspec

```diff
@@ -38,5 +38,39 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<hiera>, [">= 2.0", "< 4"])
   # PUP-7115 - return to a gem dependency in Puppet 5
   # s.add_runtime_dependency(%q<semantic_puppet>, ['>= 0.1.3', '< 2'])
+  # i18n support (gettext-setup and dependencies)
   s.add_runtime_dependency(%q<gettext-setup>, [">= 0.10", "< 1"])
+  s.add_runtime_dependency(%q<locale>, ["~> 2.1"])
+  s.add_runtime_dependency(%q<rake>, ["10.1.1"])
```

Comment:
I think you are correct, rake is usually used as a dev dep, so this could be changed.